client/my-sites/checkout/composite-checkout/wpcom/components/wp-checkout.js

```diff
@@ -226,6 +235,18 @@ const CheckoutSummaryUI = styled( CheckoutSummary )`
 	}
 `;
 
+const UpsellWrapperUI = styled.div`
+	margin-top: 1.5em;
```

Comment:
Small change, but can we use `24px` to keep consistent with the left margin of `CheckoutSummaryArea`?